salt/modules/napalm_mod.py

```diff
@@ -1105,8 +1160,13 @@ def pyeapi_conn(**kwargs):
         res1 = conn.run_commands('show version')
         res2 = conn.get_config(as_string=True)
     """
-    pyeapi_kwargs = pyeapi_nxos_api_args(**kwargs)
-    return __salt__["pyeapi.get_connection"](**pyeapi_kwargs)
+    salt.utils.versions.warn_until(
+        "Chlorine",
+        "This 'napalm_mod.pyeapi_conn' function as been deprecated and "
+        "its functionality removed, as such, it has been made an internal "
```

Comment:
```suggestion
        "will be removed in the {version} release, as such, it has been made an internal "
```

Reply:
So much for C & P from other uses of salt.utils.versions.warn_until :+1:

Reply:
Updated other usage of it  too